core/templates/dev/head/pages/admin/admin.html

```diff
@@ -429,10 +68,24 @@
 
     {% include 'components/forms/form_builder_templates.html' %}
     {% include 'pages/footer_js_libs.html' %}
+    {% include 'pages/admin/admin_navbar_directive.html' %}
+    {% include 'pages/admin/activities_tab/admin_dev_mode_activities_tab_directive.html' %}
+    {% include 'pages/admin/activities_tab/admin_prod_mode_activities_tab_directive.html' %}
+    {% include 'pages/admin/config_tab/admin_config_tab_directive.html' %}
+    {% include 'pages/admin/jobs_tab/admin_jobs_tab_directive.html' %}
+    {% include 'pages/admin/misc_tab/admin_misc_tab_directive.html' %}
 
     <script src="{{TEMPLATE_DIR_PREFIX}}/app.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/pages/Base.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/pages/admin/Admin.js"></script>
+    <script src="{{TEMPLATE_DIR_PREFIX}}/pages/admin/AdminNavbarDirective.js"></script>
+    <script src="{{TEMPLATE_DIR_PREFIX}}/pages/admin/AdminRouterService.js"></script>
+    <script src="{{TEMPLATE_DIR_PREFIX}}/pages/admin/AdminTaskManagerService.js"></script>
+    <script src="{{TEMPLATE_DIR_PREFIX}}/pages/admin/activities_tab/AdminDevModeActivitiesTabDirective.js"></script>
```

Comment:
I don't think restricting it in the frontend is quite as important. I'm going to update the backend controller to actually make it impossible to send the request in production, regardless of what's showing up in the frontend. Also, in terms of performance I don't think Angular will download the HTML for the directive if it doesn't instantiate the directive (this will be more beneficial when we're not including the HTML directly in the directive). There's no way to avoid loading the JS for the directive except via Jinja, and I don't want to proliferate using Jinja anywhere.